src/game_frontend/tailwind.config.js

```diff
@@ -26,5 +26,34 @@ export default {
       ...defaultTheme.screens
     }
   },
-  plugins: [],
+  plugins: [
+    function ({ addUtilities }) {
```

Comment:
Add this plugin by making a folder in utils directory : 
`utils/tailwind-plugins/index.tsx`